static_src/components/app_list.jsx

```diff
@@ -0,0 +1,91 @@
+
+import React from 'react';
+import Reactable from 'reactable';
+
+import SpaceStore from '../stores/space_store.js';
+
+var Table = Reactable.Table,
+    unsafe = Reactable.unsafe;
+
+function stateSetter(props) {
+  var space = SpaceStore.get(props.initialSpaceGuid);
+  return {
+    apps: space && space.apps || [],
+    currentOrgGuid: props.initialOrgGuid,
+    currentSpaceGuid: props.initialSpaceGuid
+  }
+}
+
+export default class AppList extends React.Component {
```

Comment:
Yea I could see that, and we will have more lists.

So there will be a component for each list, like applist, service list, who's responsibility will be to get and manage the data(the apps or services) and pass down to a basic list component. I'm not sure I want an inheritance model here as react usually recommends composition. 

The one issue that I'd have to figure out is some lists have slightly different ways to render columns and rows so I might have abstract that out somehow.